### ISSUE TITLE
[$40] Cannot load user profile because of missing property

### ISSUE THREAD

**Issue by ThomasKranitsas** [topcoder-platform/community-app#1911]
**Describe the bug**
The user profile is not loading for copilot users.

**To Reproduce**
1. Go to https://www.topcoder.com/members/thomaskranitsas
2. See error

**Expected behavior**
The page should load and hide the information that's missing.

**Screenshots**
<img width="1313" alt="screenshot 2019-01-29 at 11 00 35" src="https://user-images.githubusercontent.com/4612921/51896480-3d89bb00-23b5-11e9-94a9-2b686aa5bed5.png">

**Comment by Gauravseta** [comment]
@ThomasKranitsas is this open for pickup?

**Comment by ThomasKranitsas** [comment]
No, I'm working on it.. I've almost completed it. It's just a payment for myself

**Comment by Gauravseta** [comment]
@ThomasKranitsas ok

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30081378 has been updated - it has been assigned to thomaskranitsas.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```
